.github/workflows/ci.yml

```diff
@@ -56,9 +56,10 @@ env:
   #
   #  GITHUB_REGISTRY_WAIT_FOR_IMAGE: "false"
   #
+  # You can also switch back to building images locally and disabling the "Build Images" workflow
+  # by defining AIRFLOW_GITHUB_REGISTRY_WAIT_FOR_IMAGE secret with value set to "false"
 
-  GITHUB_REGISTRY_WAIT_FOR_IMAGE: "true"
-
+  GITHUB_REGISTRY_WAIT_FOR_IMAGE: ${{ secrets.AIRFLOW_GITHUB_REGISTRY_WAIT_FOR_IMAGE != 'false' }}
```

Comment:
```suggestion
  GITHUB_REGISTRY_WAIT_FOR_IMAGE: ${{ secrets.AIRFLOW_GITHUB_REGISTRY_WAIT_FOR_IMAGE }}
```

Reply:
As explained below - I only want the secret to trigger the "BUILD_IMAGE=false" if it is defined and it has "false" value. So the condition is actually OK :)

Reply:
I pushed back the original change